nautobot_device_lifecycle_mgmt/navigation.py

```diff
@@ -169,6 +169,20 @@
                 name="Reports",
                 weight=100,
                 items=(
+                    NavMenuItem(
+                        link="plugins:nautobot_device_lifecycle_mgmt:hardwarenotice_device_report",
+                        name="Device Hardware Notice - Report",
+                        permissions=[
```

Comment:
This permission should match the notice results model.

Reply:
Fixed